.github/workflows/ci.yml

```diff
@@ -130,7 +157,11 @@ jobs:
           ./quarkus-dev-cli version
       - name: Build with Maven
         run: |
-          mvn -fae -V -B --no-transfer-progress clean verify -Dinclude.quarkus-cli-tests -Dts.quarkus.cli.cmd="${PWD}/quarkus-dev-cli"${{ matrix.module-mvn-args }} -am -DexcludedGroups=long-running
+          mvn -fae -V -B --no-transfer-progress clean verify -Dinclude.quarkus-cli-tests -Dts.quarkus.cli.cmd="${PWD}/quarkus-dev-cli" ${{ matrix.module-mvn-args }} -am -DexcludedGroups=long-running
+          # Long running tests
+          if [[ -n "${{ matrix.module-long-run-mvn-args }}" ]]; then
+            mvn -fae -V -B --no-transfer-progress verify -Dinclude.quarkus-cli-tests -Dts.quarkus.cli.cmd="${PWD}/quarkus-dev-cli" ${{ matrix.module-long-run-mvn-args }} -am
```

Comment:
useless thoughts: This mean you will have 2 test executions runs in one workflow, so it's fine in regards of failsafe reports, we can ignore loss of a flaky test report in such a rare case when this is run, ok, but there is still a readability. I always scroll down and read results, now you will need to go through long logs to find first results or have a search term like `build success`.

questions:

1.  wouldn't it be clearer if you created next step that is only executed when `matrix.module-long-run-mvn-args`  is set?
2. are there any long running CLI tests or do you think it is likely there will be? it would make this command shorter if you dropped CLI stuff

I'll leave you to decide on both topics, just want to make sure you thought about that and current state is what you prefer.

Reply:
1. I think it can be possible solution - single step for long running tests and all modules run
2. I don't expect CLI tests will have long waiting condition, so I think we can drop it for now